Orochi/OrochiUtils.cpp

```diff
@@ -381,24 +381,45 @@ struct OrochiUtilsImpl
 	}
 };
 
-char* OrochiUtils::s_cacheDirectory = "./cache/";
+OrochiUtils::OrochiUtils() 
+{
+	m_cacheDirectory = "./cache/";
+}
 
-oroFunction OrochiUtils::getFunctionFromFile( oroDevice device, const char* path, const char* funcName, std::vector<const char*>* optsIn )
+OrochiUtils::~OrochiUtils() 
 {
-	const std::string cacheName = OrochiUtilsImpl::getCacheName( path, funcName );
+}
+
+oroFunction OrochiUtils::getFunctionFromFile( oroDevice device, const char* path, const char* funcName, std::vector<const char*>* optsIn )
+{ 
+	std::string cacheName = OrochiUtilsImpl::getCacheName( path, funcName );
```

Comment:
[Minor] Suggestion: Enforce constant correctness and auto type specifier when the return type can be derived from the function prototype.

```suggestion
	const auto cacheName = OrochiUtilsImpl::getCacheName( path, funcName );
```